merge_gtfs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Define the paths to the required files and directories
+# Replace these paths with the correct paths on your system
+REFERENCE_GTF="/home/amaguza/data/human_reference_genome/gencode.v44.chr_patch_hapl_scaff.annotation.gtf"
```

Comment:
For the paths, you need to use an anonymised path. For example, you copy all your files to your GitHub repo in a folder called 'data'. For example: 

"../data/human_reference_genome/etcetcetc"